lib/krane/statsd.rb

```diff
@@ -2,11 +2,11 @@
 require 'statsd-instrument'
 require 'logger'
 
-module KubernetesDeploy
+module Krane
   class StatsD
     extend ::StatsD
 
-    PREFIX = "KubernetesDeploy"
+    PREFIX = "Krane"
```

Comment:
I don't think this should change yet--it should match the name of the gem the user is installing

Reply:
Oh, this makes sense! Fixing it asap 👍